src/api/index.js

```diff
@@ -267,9 +267,13 @@ export const createImage = ({
   description,
   release,
   architecture,
-  imageType,
+  imageType: imageTypes,
   'selected-packages': packages,
 }) => {
+  let imageType = imageTypes[0];
```

Comment:
I'd use array destruct here
```suggestion
  let [ imageType ] = imageTypes || [];
```